packages/@aws-cdk/aws-dynamodb/lib/replica-handler/index.ts

```diff
@@ -0,0 +1,71 @@
+/* tslint:disable no-console */
+import { DynamoDB } from 'aws-sdk'; // eslint-disable-line import/no-extraneous-dependencies
+import { execSync } from 'child_process';
+
+let latestSdkInstalled = false;
+
+function installLatestSdk(): void {
+  console.log('Installing latest AWS SDK v2');
+  // Both HOME and --prefix are needed here because /tmp is the only writable location
+  execSync('HOME=/tmp npm install aws-sdk@2 --production --no-package-lock --no-save --prefix /tmp');
+  latestSdkInstalled = true;
+}
+
+export async function onEventHandler(event: any): Promise<any> {
```

Comment:
@eladb I was not able to use the framework types (`AWSCDKAsyncCustomResource`) exported in `@aws-cdk/custom-resources`... any idea why? or how I could use them?

Reply:
If I export those types in `lib/provider-framework/index.ts` (`export * from './types'`), I can use them but then `jsii` complains when building `@aws-cdk/custom-resources` because types are extending types defined in `@types/aws-lambda`

Reply:
Of course it works by deeply importing (`import { IsCompleteResponse, OnEventResponse } from '@aws-cdk/custom-resources/lib/provider-framework/types';`), this is OK inside this repo but not a good users experience for user consuming `@aws-cdk/custom-resources`